src/printer/smt2/smt2_printer.cpp

```diff
@@ -2044,45 +2045,16 @@ static void toStream(std::ostream& out, const EchoCommand* c, Variant v)
    --------------------------------------------------------------------------
 */
 
-static void toStream(std::ostream& out, const SynthFunCommand* c)
+static void toStream(std::ostream& out, const Type& t)
```

Comment:
This method should be named `toStreamSygusGrammar`, since it's not printing a type in its normal sense.

Reply:
Done!